src/wp-includes/rest-api/endpoints/class-wp-rest-controller.php

```diff
@@ -581,6 +581,18 @@ public function get_fields_for_response( $request ) {
 
 		$fields = array_keys( $properties );
 
+		/*
+		 * '_links' and '_embedded' are not typically part of the item schema,
+		 * but they can be specified in '_fields', so they are added here as a
+		 * convenience for checking with rest_is_field_included().
+		 */
+		$fields[] = '_links';
```

Comment:
fields support passing sub-fields like this. ?_fields=_links.author. For consistency we should also support that here .Thoughts @TimothyBJacobs

Reply:
Sorry, could you describe a little more what you have in mind?

Reply:
@dlh01 So the fields param in the URL supports sub fields. So you can pass, `?_fields=excerpt.protected` it will not get all other fields of excerpt and only return `protected`. Example - https://spacedmonkey.com/wp-json/wp/v2/posts?_fields=excerpt.protected

So do we support this - `?_fields=_links.self` and only prepare self.

Reply:
Got it, thanks. Currently, I've opted to not check for each link, simply to avoid the complexity of modifying the `prepare_links()` signature on each controller to accept the requested fields or the request object. This question was also discussed in Slack when the ticket was opened: https://wordpress.slack.com/archives/C02RQC26G/p1617223839028400.

Reply:
Thanks for your answer. That is great to see this has already be thought of.